src/bolt/utils.ts

```diff
@@ -59,55 +63,123 @@ export async function getReactionData (event: ReactionAddedEvent | ReactionRemov
 
     const getMessageInfo = (data: ConversationsRepliesResponse): ReactionData => {
         const message = (data.messages ?? [])[0] as GenericMessageEvent;
+        let pullRequestLink = '';
+
+        // Try to parse github pull request from text message if available.
+        if (message.text) {
+            const match = /https:\/\/github.com\/[^ ]+?\/pull\/\d+/.exec(message.text);
+            pullRequestLink = match ? match[0] : '';
+        }
+
+        // If there is no PR info, try to check the message attachments, message is coming from github event.
+        if (!pullRequestLink.length && message.attachments?.length) {
+            const {title} = message.attachments[0];
+            if (title) {
+                const match = /https:\/\/github.com\/[^ ]+?\/pull\/\d+/.exec(title);
```

Comment:
Could consolidate this regex in a const rather than having to re-create it in multiple places.

Reply:
Great idea, will add that in the next improvement changes.